web/styles/right_sidebar.css

```diff
@@ -363,38 +363,55 @@ $user_status_emoji_width: 24px;
     grid-template-rows: var(--line-height-sidebar-row-prominent);
     grid-template-columns: minmax(0, 1fr) auto;
     align-items: center;
-    margin-right: var(--width-simplebar-scroll-hover);
+    margin-bottom: 10px;
 
     #userlist-header-search {
         display: grid;
         grid-template-rows: var(--line-height-sidebar-row-prominent);
-        grid-template-columns: minmax(0, 1fr) 20px;
+        grid-template-columns: minmax(0, 1fr) 30px;
         align-items: center;
-    }
 
-    #userlist-title {
-        margin: 0;
-    }
+        & .user-list-filter {
+            grid-area: 1 / 1 / 2 / 3;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow: hidden;
+            /* Prevent text from colliding with #clear_search_button */
+            padding-right: 28px;
+            /* Push back against inherited styles; let CSS Grid be in
+               charge of the height. */
+            height: auto;
+        }
 
-    #user_filter_icon {
-        opacity: 0.5;
-        justify-self: center;
+        #clear_search_people_button {
+            grid-area: 1 / 2 / 2 / 3;
+            padding: 0;
+            background: none;
+            color: var(--color-text-clear-search-button);
+            display: grid;
```

Comment:
Why is this being displayed as a grid? Not part of the PR, but I'm confused about that on `#buddy-list-menu-icon` below, too.

Reply:
I was trying to get the clear button centered and this is what I ended up doing, but if you have other ideas I'd be interested in that.

Reply:
Okay. I think this is all right for now, but I've also made a note to look into this more closely.